packages/database/lib/migrations/20250418151811_plan_add_max_connection.cjs

```diff
@@ -0,0 +1,13 @@
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function (knex) {
+    await knex.raw(`ALTER TABLE "plans" ADD COLUMN "connections_max" int8`);
+};
+
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.down = async function (knex) {
+    await knex.raw(`ALTER TABLE "plans" DROP COLUMN "connections_max"`);
+};
```

Comment:
I thought we gave up on the `down` 😜

Reply:
Not sure we officially agreed, but if we shake hands really happy to stop!